### PR TITLE
Made changes to the UI of footer and sponsors section

### PR DESCRIPTION
**Pull request by khush2706**
# Changed Footer UI and Sponsors section

## Description

- Changed the layout of sponsors section and made it responsive.
- Swizzled the footer code.
- Changed the layout of the footer section.
- Added social icons for links in the footer.
- Made footer Responsive.
- Removed irrelevant links from the footer.

Fixes # (issue)
#228 

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue).

...

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by sohamsshah** [comment]
Hey @khush2706 , can you please post screenshots of the UI changes done?

**Comment by khush2706** [comment]
> Hey @khush2706 , can you please post screenshots of the UI changes done?

I have changed the footer UI from this:
Desktop view

Mobile view

To this:
image

image

And the mobile view of Sponsors section from this:
image

To this:
image

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -179,34 +171,49 @@ module.exports = {
...
         {
-          title: 'Social',
+          title: 'Follow Us',
           items: [
             {
-              label: 'Twitter ',
-              href: 'https://twitter.com/mojaglobal',
+              html: `
+              <a href= 'https://twitter.com/mojaglobal'>
+              <img class="socials" src="/img/twitter.png" />
```
Comment (sohamsshah) [inline website/docusaurus.config.js:186]: Can we have `alt` attribute on the img tags for better accessibility?

File: website/src/theme/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```
Comment (sohamsshah) [inline website/src/theme/Footer/index.js:2]: Remove these comment lines relating to Facebook Inc.

File: website/src/theme/Footer/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```
Comment (sohamsshah) [inline website/src/theme/Footer/styles.module.css:2]: Please remove these comments as well

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Hey @khush2706 , thanks for working on this! I have suggested some minor changes in the implementation.

Few changes: 
image4

Can we remove the horizontal line and increase the font-size back to the previous size to the `copyright` text?

- The icons under the `Follow Us` title has a small left margin which leads the icons to misalign with the title text. Should be a simple fix.

After that this PR is good to be merged. Feel free to ping if you have any blockers.

**Comment by khush2706** [comment]
@sohamsshah I have made the suggested changes. Kindly check it.

**Review by sohamsshah — APPROVED** [review]
LGTM! Thanks @khush2706 for the contribution! 🎉

**Comment by HarshCasper** [comment]
@all-contributors bot please add @khush2706 for code

**Pull request merged**